app/views/dvr/dvr_files.scala.html

```diff
@@ -23,8 +23,11 @@
 
 @(model: AvailableRequestDetailedValuation, submissionId: String, owner: Boolean, authorisationId: Long, backUrl: String, startCheckUrl: String, checkCases: List[CaseDetails], challengeCases: List[CaseDetails])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
-@paperChallengeUrl = @{
-    controllers.routes.Assessments.startChallengeFromDVR(submissionId, model.valuationId, model.uarn, owner).url
+@challengeFromBackLinkUrl = @{
```

Comment:
maybe change this to "returnUrl"? (i think it reads better challengeFromBackLinkUrl)